build.gradle

```diff
@@ -3,27 +3,23 @@ plugins {
 	id 'eclipse'
 	id 'idea'
 	id 'maven-publish'
-	id 'fabric-loom' version '0.6-SNAPSHOT'
-	id 'net.minecrell.licenser' version '0.4.1'
+	id 'fabric-loom' version "0.7-SNAPSHOT"
+	id 'net.minecrell.licenser' version "0.4.1"
```

Comment:
licenser seems to break checks, maybe this needs updating as well?
the latest is `0.6.0`

Reply:
It didn't work. It says it wasn't found. :(

Reply:
looks like the id got updated as well
its `org.cadixdev.licenser` now